src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,118 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class CalculatorTest {
+
+    @DisplayName("피연산자가 숫자인지 체크")
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "29", "10000", "2222"})
+    public void validateOperand1(String operand) throws Exception {
+        //then
+        Calculator.validateOperand(operand);
+    }
+
+    @DisplayName("피연산자가 숫자인지 체크 - fail 체크")
+    @ParameterizedTest
+    @ValueSource(strings = {"1ㅁ", "2d", "d", "-", "29`", "100~00", "^"})
+    public void validateOperand2(String operand) throws Exception {
+        //then
+        assertThatThrownBy(() -> {
+            Calculator.validateOperand(operand);
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("연산자가 사칙연산인지 체크")
+    @ParameterizedTest
+    @ValueSource(strings = {"+", "-", "*", "/"})
+    public void validateOperator1(String operator) throws Exception {
+        //then
+        Calculator.validateOperator(operator);
+    }
+
+    @DisplayName("연산자가 사칙연산인지 체크")
+    @ParameterizedTest
+    @ValueSource(strings = {"++", "/-", "--", "***", "1", "ㅁ", "a"})
+    public void validateOperator2(String operator) throws Exception {
+        //then
+        assertThatThrownBy(() -> {
+            Calculator.validateOperator(operator);
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    /**
+     * input 2개와 결과값 1개 총 3개가 필요함
+     * CsvSource 를 이용할 경우 delimiter로 2개의 값으로 분리가 가능한데
+     * 3개의 값을 자동으로 테스트 하는 방법은 없을까요?
+     */
+    @DisplayName("덧셈 테스트")
+    @Test
+    public void plus() throws Exception {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    @ParameterizedTest
    @CsvSource(value = {"1,2,3"})
    public void plus(double operand1, double operand2, double result) throws Exception {
```